index.js

```diff
@@ -20,13 +21,45 @@ function configureBot(bot) {
   let farmingInProgress = false;
   let farmingDeliveryRun = false;
 
+  let matchInfo = null;
+
+
+  /**
+   * Handle match info updates
+   */
+  matchInfoEmitter.on('score_update', (matchInfo) => {
+    console.log(`Match scores updated`)
+    this.matchInfo = matchInfo;
+  })
+
+  matchInfoEmitter.on('match_started', (matchInfo) => {
+    console.log(`The match has started`)
+    this.matchInfo = matchInfo;
+  })
+
+  matchInfoEmitter.on('match_ended', (matchInfo) => {
+    console.log(`The match has ended`)
+    this.matchInfo = matchInfo;
+  })
+
+  matchInfoEmitter.on('player_joined', (matchInfo, username, team) => {
+    console.log(`${username} on team: ${team} has joined the match `)
+    this.matchInfo = matchInfo;
+  })
+
+  matchInfoEmitter.on('player_left', (matchInfo, username, team) => {
+    console.log(`${username} on team: ${team} has left the match `)
+    this.matchInfo = matchInfo;
+  })
+
```

Comment:
Not needed right now, but we could probably have have a generic version of this for people who don't need to respond to individual events, i.e.

```javascript
let matchInfo = null;
matchInfoEmitter.saveUpdatesTo(matchInfo)
```

Reply:
(here matchInfoEmitter is some subclass of EventEmitter)